scripts/build.mjs

```diff
@@ -0,0 +1,87 @@
+import esbuild from "esbuild";
+import { copyFile, readFile, writeFile, rm } from "fs/promises";
+import { glob } from "glob";
+
+const sharedOptions = {
+  sourcemap: "external",
+  sourcesContent: false,
```

Comment:
```suggestion
  sourcesContent: true,
```
> Let's just make it output the same as Pika for now, and we can revisit these points later

I'm sorry here, I don't know what I was checking but the `sourceContent` is served in Pika so I think we should be keeping it for ESBuild for the moment.